installer.sh

```diff
@@ -135,11 +135,11 @@ create_user_linux() {
 	# TODO: Do we need others?
 	SUDOERS_CONTENT=$(cat <<- EOF
 		$username	ALL=(ALL)	ALL
-		$username	ALL=(ALL)	NOPASSWD: /usr/bin/shutdown -h now, /usr/bin/halt, /usr/bin/poweroff, /usr/bin/reboot
-		$username	ALL=(ALL)	NOPASSWD: /usr/bin/systemctl start, /usr/bin/systemctl stop
-		$username	ALL=(ALL)	NOPASSWD: /usr/bin/mount -o nosuid\,nodev\,noexec, /usr/bin/umount
+		$username	ALL=(ALL)	NOPASSWD: /sbin/shutdown -h now, /sbin/halt, /sbin/poweroff, /sbin/reboot
+		$username	ALL=(ALL)	NOPASSWD: /bin/systemctl start, /bin/systemctl stop
+		$username	ALL=(ALL)	NOPASSWD: /bin/mount -o nosuid\,nodev\,noexec, /bin/mount, /bin/umount
```

Comment:
Which options do you need for mount? I would rather just allow a couple of them instead of just everything.

Reply:
Here are the different mount commands from backitup

https://github.com/simatec/ioBroker.backitup/blob/043e36a62b87c6dd83e477fdde75d808795fdc1a/lib/scripts/01-mount.js#L58

https://github.com/simatec/ioBroker.backitup/blob/043e36a62b87c6dd83e477fdde75d808795fdc1a/lib/scripts/01-mount.js#L81

https://github.com/simatec/ioBroker.backitup/blob/043e36a62b87c6dd83e477fdde75d808795fdc1a/lib/scripts/01-mount.js#L43